src/morphocut/image.py

```diff
@@ -271,6 +280,16 @@ def transform_stream(self, stream):
 @ReturnOutputs
 @Output("image")
 class Gray2RGB(Node):
+    """
+    Converts the Grayscale image to RGB image.
+
+    .. note::
+        Uses the skimage module :py:class:`skimage.color` to convert from Grayscale to RGB.
```

Comment:
The reference is not correct. It should be:
```
:py:func:`skimage.color.gray2rgb`
```

Please check the rendering of the documentation when you use references.